web/concrete/src/Config/Repository/Repository.php

```diff
@@ -56,6 +57,11 @@ public function save($key, $value)
         if ($this->saver->save($item, $value, $this->environment, $group, $namespace)) {
             $this->load($group, $namespace, $this->getCollection($group, $namespace));
 
+            switch ($key) {
+                case 'concrete.seo.url_rewriting':
+                    Core::make('helper/file/htaccess')->setCurrentRewriteRules((bool) $value);
+                    break;
+            }
```

Comment:
I thought that this was what you meant with your [comment on #3123](https://github.com/concrete5/concrete5/pull/3123#issuecomment-156530081)